sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -30,7 +30,7 @@ import {
 } from "./generatedModels";
 import { AbortSignalLike } from "@azure/abort-controller";
 import { BlobDownloadResponse } from "./BlobDownloadResponse";
-import { Blob } from "./generated/src/operations";
+import { Blob as StorageBlob } from "./generated/src/operations";
```

Comment:
Aliasing `Blob` from codegen as `StorageBlob` to resolve a conflict with the `Blob` type from codegen and the `Blob` type in the browser. 

**Conflict** 
```typescript
  // member of BlockBlobClient
  private async uploadSeekableBlob(
    blobFactory: (offset: number, size: number) => Blob, 
    size: number,
    options: BlockBlobParallelUploadOptions = {}
  )
```
**Fix** 
![image](https://user-images.githubusercontent.com/10452642/67448991-8f224780-f5cd-11e9-8f6f-9bb3fa64028f.png)

Reply:
This is only used internally and not exposed to the public. So, maybe fine?